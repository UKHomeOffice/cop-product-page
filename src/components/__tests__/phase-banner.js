import React from "react"
import renderer from "react-test-renderer"
import PhaseBanner from "../phase-banner";

describe("PhaseBanner", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<PhaseBanner/>)
      .toJSON();
    expect(tree).toMatchSnapshot()
  })
})