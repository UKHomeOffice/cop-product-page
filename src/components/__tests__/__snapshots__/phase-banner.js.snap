// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhaseBanner renders correctly 1`] = `
<div
  className="govuk-phase-banner"
  style={
    Object {
      "marginBottom": "40px",
    }
  }
>
  <p
    className="govuk-phase-banner__content"
  >
    <strong
      className="govuk-tag govuk-phase-banner__content__tag"
    >
      beta
    </strong>
    <span
      className="govuk-phase-banner__text"
    >
      This is a new service â€“ your
       
      <a
        className="govuk-link"
        href="https://gov.uk"
      >
        feedback
      </a>
       
      will help us to improve it.
    </span>
  </p>
</div>
`;
